### PR TITLE
Feature/set

### PR DESCRIPTION
**Pull request by eplumite**
Function set haven't been implemented yet.

### CODE REVIEW

**Comment by NevVerVer** [comment]
The function and the test done completely and fulfill the requirements in the task :)

File: storage.py [hunk 1]
```diff
@@ -15,8 +15,9 @@ def get(self, key):
     def remove(self):
         pass
 
-    def set(self):
-        pass
+    def set(self, key, value):
+        if key in self.data:
```
Comment (nosmokingsurfer) [inline storage.py:19]: The set method should rise an exception KeyError. There was a similar questions in telegram chat;
Screenshot from 2020-10-04 22-09-19

**Review by nosmokingsurfer — COMMENTED** [review]
Everything looks OK

**Review by NevVerVer — APPROVED** [review]
everything done well

**Review by nosmokingsurfer — APPROVED** [review]
ok

**Pull request merged**